cmd/xl-sets.go

```diff
@@ -1488,7 +1488,9 @@ func (s *xlSets) HealFormat(ctx context.Context, dryRun bool) (res madmin.HealRe
 		s.format = refFormat
 
 		// Disconnect/relinquish all existing disks, lockers and reconnect the disks, lockers.
+		s.xlDisksMu.Lock()
 		s.xlDisks.Close()
+		s.xlDisksMu.Unlock()
```

Comment:
we used to have lock statements before right?

Reply:
Yes it was changed for some reason @krishnasrinivas